### ISSUE TITLE
a11y: va-accordion

### ISSUE THREAD

**Issue by kushich** [epicmaxco/vuestic-ui#3288]
- [x] step 1: investigate a11y guides for this component;
- [x] step 2: discuss with team;
- [ ] step 3: implement required functionality;
- [ ] step 4: add docs page section about a11y.

**Comment by kushich** [comment]
Check if we use ARIA attributes such as `role`, `aria-expanded`, and `aria-controls` to provide additional information to assistive technologies. These attributes help screen readers to announce the state of the accordion and its panels.
#### Keyboard navigation ([ref](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/))
| Key | Description |
|---|---|
| Enter or Space | - When focus is on the accordion header for a collapsed panel, expands the associated panel. If the implementation allows only one panel to be expanded, and if another panel is expanded, collapses that panel.  <br/> - When focus is on the accordion header for an expanded panel, collapses the panel if the implementation supports collapsing. Some implementations require one panel to be expanded at all times and allow only one panel to be expanded; so, they do not support a collapse function. |
 |Tab|Moves focus to the next focusable element; all focusable elements in the accordion are included in the page Tab sequence.|
 |Shift + Tab|Moves focus to the previous focusable element; all focusable elements in the accordion are included in the page Tab sequence.|
 
 #### WAI ARIA
- `VaCollapse`'s header should have role [button](https://w3c.github.io/aria/#button).
- Each `VaCollapse`'s header is wrapped in an element with role [heading](https://w3c.github.io/aria/#heading) that has a value set for [aria-level](https://w3c.github.io/aria/#aria-level) that is appropriate for the information architecture of the page.
- If the content of `VaCollapse` is visible, header of `VaCollapse` element has [aria-expanded](https://w3c.github.io/aria/#aria-expanded) set to true. If the panel is not visible, [aria-expanded](https://w3c.github.io/aria/#aria-expanded) is set to false.
- `VaCollapse` header element has [aria-controls](https://w3c.github.io/aria/#aria-controls) set to the ID of the element containing the accordion panel content.
- If the `VaCollapse` content is visible, and if the accordion does not permit it to be collapsed, its header has [aria-disabled](https://w3c.github.io/aria/#aria-disabled) set to true.
 #### Questions to discuss
 - Should we expand/collapse our accordion item of focus?
 - Should we configure levels for `VaCollapse` headers (nested collapses)?

**Comment by asvae** [comment]
> Should we expand/collapse our accordion item of focus?

Depending on `multiple` prop.

> Should we configure levels for VaCollapse headers (nested collapses)?

Needs more data - we don't know.

**Comment by xiongmao86** [comment]
Can I take a stab of this?

**Comment by xiongmao86** [comment]
@asvae does the team make decision on whether to configure aria-level or not?
